src/SFA.DAS.EmployerIncentives.Api/Controllers/ApplicationController.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Api.Models;
+using SFA.DAS.EmployerIncentives.Application.Queries.GetApplication;
+using SFA.DAS.EmployerIncentives.Models;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [ApiController]
+    public class ApplicationController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<ApplicationController> _logger;
+
+        public ApplicationController(IMediator mediator, ILogger<ApplicationController> logger)
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        [HttpGet]
+        [Route("/accounts/{accountId}/applications/{applicationId}")]
```

Comment:
hmm i thought `/accounts/{accountId}/` is being automatically added somewhere :/